rclrs/src/node/publisher.rs

```diff
@@ -107,6 +108,30 @@ where
         })
     }
 
+    /// Returns the topic that the publisher is publishing data to
```

Comment:
Could you add documentation in the same style as [the one for `Node::name` etc](https://github.com/ros2-rust/ros2_rust/blob/30b193152e27c09cf9101265d94cc10358edd58e/rclrs/src/node.rs#L122-L143)? E.g.

```suggestion
    /// Returns the topic name of the publisher.
    ///
    /// This returns the topic name after remapping, so it is not necessarily the same as the
    /// topic name that was used when creating the publisher.
```

It's okay to not have example code, though (because we'd need an extra dependency on a message package, which we don't want).

Reply:
These comments of course also apply to the subscription.

Reply:
The comments have been formatted as requested... Please do let me know if there are any further changes required